runners/flink/runner/src/main/java/org/apache/beam/runners/flink/translation/FlinkBatchTransformTranslators.java

```diff
@@ -302,25 +301,8 @@ public void translateNode(Write.Bound<T> transform, FlinkBatchTranslationContext
     }
   }
 
-  private static class GroupByKeyOnlyTranslatorBatch<K, V> implements FlinkBatchPipelineTranslator.BatchTransformTranslator<GroupByKey.GroupByKeyOnly<K, V>> {
```

Comment:
I removed this, but I admit to not being sure what the intent is. There is a translator for the whole `GroupByKey` so this should have been dead code. On the other hand, the translator translates `GroupByKey` to `GroupByKeyOnly` so perhaps it would be better to use the expanded form, like Spark and the `DirectPipelineRunner`.

LMK